.rubocop_todo.yml

```diff
@@ -1,107 +1,78 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2017-10-11 09:34:35 +0100 using RuboCop version 0.50.0.
+# on 2018-09-18 16:02:28 +0100 using RuboCop version 0.59.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 234
+# Cop supports --auto-correct.
+Layout/EmptyLineAfterGuardClause:
+  Enabled: false
```

Comment:
Is this a temp fix?  If not, shouldn't it be in `rubocop.yml` or just autocorrected?

Reply:
not keen on this cop as a stlye issue. as discussed will move into rubocop